.travis.yml

```diff
@@ -9,12 +9,21 @@ python:
   - "3.6"
 
 matrix:
- include:
-  - python: 3.6
-    env: TOXENV=i18n
+  include:
```

Comment:
The indentation in this file was inconsistent. The Python versions were indented with two spaces, but everything else was indented with one space.

I opted to indent everything two spaces. Happy to revert these particular changes if they're objectionable for some reason.